mobile/lib/ui/sharing/manage_links_widget.dart

```diff
@@ -318,6 +330,52 @@ class _ManageSharedLinkWidgetState extends State<ManageSharedLinkWidget> {
     );
   }
 
+  Future<Uint8List> _createAlbumPlaceholder(
```

Comment:
Since `_createAlbumPlaceholder` and `_shareAlbumLink` is written twice in both on `manage_links_widget.dart` and `share_collection_page.dart`, let move both to `share_util.dart` to avoid duplicate code. Can rename `shareAlbumLink` to `shareAlbumLinkWithPlaceholder` in `share_util.dart` and call it from both files.